pkgs/applications/misc/keystore-explorer/default.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchzip, stdenv, jre }:
+
+stdenv.mkDerivation rec {
+  version = "5.4.4";
+  name = "keystore-explorer-${version}";
+  src = fetchzip {
+    url = "https://github.com/kaikramer/keystore-explorer/releases/download/v${version}/kse-544.zip";
+    sha256 = "01kpa8g6p6vcqq9y70w5bm8jbw4kp55pbywj2zrhgjibrhgjqi0b";
+  };
+
+  buildInputs = [ jre ];
+
+  installPhase = ''
+    substituteInPlace ./kse.sh --replace "java -jar" "${jre}/bin/java -jar"
+    mkdir -p $out/bin
+    cp -R ./* $out/
+    rm $out/kse.exe
+    mv $out/kse.sh $out/keystore-explorer.sh
+    ln -s $out/keystore-explorer.sh $out/bin/keystore-explorer
+  '';
+
+  meta = {
+    description = "Open source GUI replacement for the Java command-line utilities keytool and jarsigner";
+    license = stdenv.lib.licenses.gpl3;
```

Comment:
gpl3 is ambitious. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 .

Reply:
Changed to gpl3Only since keystore-explorer currently advertises GPL3 on github.